modules/tide_site_alert/src/Plugin/Block/TideSiteAlertBlock.php

```diff
@@ -29,8 +29,11 @@ public function build() {
       if ($value && strpos($value, 'site_alert:') !== FALSE) {
         preg_match('#[0-9]+$#', $value, $match);
         $site_alert = SiteAlert::load(reset($match));
-        $build[$key]['#alert']['start'] = date('d/m/Y H:i:s', strtotime($site_alert->getStartTime()));
-        $build[$key]['#alert']['end'] = date('d/m/Y H:i:s', strtotime($site_alert->getEndTime()));
+        $dateTimeImmutable = new \DateTimeImmutable();
```

Comment:
Hmm I think this might not be correct.

DateTimeImmutable should be replacing date() - you're creating a new DateTimeImmutable object to work with, to create a formatted output for your two build variables. It should probably be more like this:

```
$build[$key]['#alert']['start'] = new \DateTimeImmutable($site_alert->getStartTime())->format('d/m/Y H:i:s');
$build[$key]['#alert']['end'] = new \DateTimeImmutable($site_alert->getEndTime())->format('d/m/Y H:i:s');
```

With your code, you're creating a new object based on time now(), and then modifying it by your start times/end times.

Does this make sense?